examples/ocean-dex-bot/src/main.ts

```diff
@@ -0,0 +1,101 @@
+import { WIF } from '@defichain/jellyfish-crypto'
+import { WalletClassic } from '@defichain/jellyfish-wallet-classic'
+import { WhaleApiClient } from '@defichain/whale-api-client'
+import { MainNet } from '@defichain/jellyfish-network'
+import { WhaleWalletAccount } from '@defichain/whale-api-wallet'
+import { CTransactionSegWit, TransactionSegWit } from '@defichain/jellyfish-transaction'
+import { BigNumber } from 'bignumber.js'
+import { AddressToken } from '@defichain/whale-api-client/dist/api/address'
```

Comment:
Is this intentionally import from the `dist` folder?

```suggestion
import { AddressToken } from '@defichain/whale-api-client'
```

Reply:
The export on `@defichain/whale-api-client` wasn't designed properly. I think you need to:

```ts
import { address } from '@defichain/whale-api-client'

private async getAddressToken (id: number): Promise<address.AddressToken | undefined> {
}
```

It wasn't properly enforced there, moving forward this project will be consolidated into jellyfish for better engineering opportunities. #977

Reply:
Noted, that ticket looks like something I wouldn't mind picking up.